client/src/components/league-teams/league-teams.html

```diff
@@ -1,5 +1,5 @@
-<div class="league">
-    <h1 class="league__title"><span data-bind="text: selectedLeagueName()"></span> teams</h1>
+<div class="container">
+    <h1 class="title"><span data-bind="text: selectedLeagueName()"></span> teams</h1>
```

Comment:
You don't have to put parenthesis everywhere in markup when you use an observable value.